### ISSUE TITLE
Failed asserting that NULL is null

### ISSUE THREAD

**Issue by wkhayrattee** [pestphp/pest#294]
Hello, first of all I am blown away by PestPHP, amazing tool.

Just one thing I encountered when I was asserting that a resource handle was null after invoking close method.
Context below.

More precisely, the test is [here](https://github.com/wkhayrattee/simple-dba/blob/ea912e4eb65619460ab0b77df39abdf47da34f5f/tests/DbaTest.php#L27) 
`expect($this->dbaObject->resource_handle)->toBeNull();`

When I call the dba_close `dba_close($this->resource_handle);`, the handle becomes NULL. and it would seem correct as PestPHP also mentions **_asserting NULL is null_**, which hints that the handle was indeed **NULL**.

So my question is, **is NULL and null same**?

Right to make this test PASS, I am having to explicitly assign null to the handle after it is close(), see [here](https://github.com/wkhayrattee/simple-dba/blob/ea912e4eb65619460ab0b77df39abdf47da34f5f/src/Dba.php#L65)
`        $this->resource_handle = null; //else pestPHP issue: Failed asserting that NULL is null`

Would like to know your thoughts, may be I am doing something wrong?

**Comment by owenvoke** [comment]
`NULL` and `null` should be the same. However, looking into this, the `$resource_handle` property isn't actually set to null by `dba_close()` (you can see this by running a `var_dump($this->dbaObject->resource_handle)` before the expectation check), and Pest is just displaying the unknown closed resource value as `NULL`. Therefore, this doesn't seem to be an issue with Pest.

I think you would need to manually set it to `null` in order for this to be correct.

**Comment by wkhayrattee** [comment]
> `NULL` and `null` should be the same. However, looking into this, the `$resource_handle` property isn't actually set to null by `dba_close()` (you can see this by running a `var_dump($this->dbaObject->resource_handle)` before the expectation check), and Pest is just displaying the unknown closed resource value as `NULL`. Therefore, this doesn't seem to be an issue with Pest.

You are correct about this. I'll create a ticket with PHP to see what they say as anyone would actually expect a proper **null**.


> I think you would need to manually set it to `null` in order for this to be correct.

Indeed, that's what I precisely did, [glance here](https://github.com/wkhayrattee/simple-dba/blob/ea912e4eb65619460ab0b77df39abdf47da34f5f/src/Dba.php#L65).


thank you for your feedback @owenvoke, cheers
